apps/astarte_appengine_api/mix.lock

```diff
@@ -3,8 +3,8 @@
   "amqp": {:hex, :amqp, "1.2.1", "eab7b4e2ce86bc84e3c2f8553e85baffd2874ae9fbab7b5c1856a731ac023f53", [:mix], [{:amqp_client, "~> 3.7.11", [hex: :amqp_client, repo: "hexpm", optional: false]}, {:goldrush, "~> 0.1.0", [hex: :goldrush, repo: "hexpm", optional: false]}, {:jsx, "~> 2.9", [hex: :jsx, repo: "hexpm", optional: false]}, {:lager, "~> 3.6.5", [hex: :lager, repo: "hexpm", optional: false]}, {:rabbit_common, "~> 3.7.11", [hex: :rabbit_common, repo: "hexpm", optional: false]}, {:ranch, "~> 1.7", [hex: :ranch, repo: "hexpm", optional: false]}, {:recon, "~> 2.3", [hex: :recon, repo: "hexpm", optional: false]}], "hexpm", "803aab6417cb142c2c24c2a1ad2433adb745a05a8cd9a19173e26933f3c1e961"},
   "amqp_client": {:hex, :amqp_client, "3.7.15", "99c46aad3406199f9aec95356f43fa73f28af2fc9da1de5b601367231e37992c", [:make, :rebar3], [{:rabbit_common, "3.7.15", [hex: :rabbit_common, repo: "hexpm", optional: false]}], "hexpm", "2dd7644cc59451f22e20cd75919e3941acb014acb4c30d9765f93ff969276750"},
   "astarte_core": {:git, "https://github.com/astarte-platform/astarte_core.git", "c332be67aad5be06fbd2cf07e89747a113b8665a", []},
-  "astarte_data_access": {:git, "https://github.com/astarte-platform/astarte_data_access.git", "ba57ad73c9a66aa6c9cd3788fb032bdcf8dd447e", []},
-  "astarte_rpc": {:git, "https://github.com/astarte-platform/astarte_rpc.git", "94e6ae73cca0de136fba54105db0ac2ad7c503a1", []},
+  "astarte_data_access": {:git, "https://github.com/astarte-platform/astarte_data_access.git", "afe0977a5b8a7a31dbb1028903328c1015543606", []},
+  "astarte_rpc": {:git, "https://github.com/astarte-platform/astarte_rpc.git", "f4bc32c47e0ddce95cc8d44e2cfe7a6b977ef7cf", []},
   "base64url": {:hex, :base64url, "0.0.1", "36a90125f5948e3afd7be97662a1504b934dd5dac78451ca6e9abf85a10286be", [:rebar], [], "hexpm", "fab09b20e3f5db886725544cbcf875b8e73ec93363954eb8a1a9ed834aa8c1f9"},
   "certifi": {:hex, :certifi, "2.5.1", "867ce347f7c7d78563450a18a6a28a8090331e77fa02380b4a21962a65d36ee5", [:rebar3], [{:parse_trans, "~>3.3", [hex: :parse_trans, repo: "hexpm", optional: false]}], "hexpm", "805abd97539caf89ec6d4732c91e62ba9da0cda51ac462380bbd28ee697a8c42"},
   "conform": {:hex, :conform, "2.5.2", "7035787a9c09d28607745444e7a1700426dc47c452634a5694033fa2fbb3414c", [:mix], [{:neotoma, "~> 1.7.3", [hex: :neotoma, repo: "hexpm", optional: false]}], "hexpm", "2056f5067e5bd4cf9d11ac093ada83c44163ad94555cadc3720fa9defaaf3bac"},
```

Comment:
I see that conform and distillery are still here, run
```
mix deps.unlock --unused
```
That should unlock deps that are not used anymore